Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -3327,15 +3342,22 @@ command revIDEOpenPalette pPaletteName
          break
       case "project browser"
       case "extension manager"
+      case "menu builder"
+      case "extension builder"
          modeless stack tStackName
          break
       default
```

Comment:
Yes, it doesn't make a difference functionally, but you're right, default as last item is better coding style.